packages/manager/apps/container/src/container/nav-reshuffle/header/user-account-menu/Content.tsx

```diff
@@ -68,20 +68,24 @@ const UserAccountMenu = ({
         className="oui-navbar-menu oui-navbar-menu_fixed oui-navbar-menu_end p-3"
         data-navi-id="account-sidebar-block"
       >
-        <h1 className="oui-heading_4 mb-1">{`${user.firstname} ${user.name}`}</h1>
+        <h1 className={`oui-heading_4 mb-1 ${isSubUser && 'text-truncate'}`}>{isSubUser ? user.auth.user : `${user.firstname} ${user.name}`}</h1>
```

Comment:
You should define properties in the component to ease reading tsx part and group subUser logic
```suggestion
        <h1 className={classNameUser}>{userFullName}</h1>
```

Reply:
I fixed